sdks/python/apache_beam/internal/json_value_test.py

```diff
@@ -67,6 +67,19 @@ def test_with_type(self):
   def test_none_from(self):
     self.assertIsNone(from_json_value(to_json_value(None)))
 
+  def test_long_value(self):
+    self.assertEquals(long(27), from_json_value(to_json_value(long(27))))
```

Comment:
Can you also test a large integer here?
 +    self.assertEquals(long(1 << 35), from_json_value(to_json_value(long(1 << 35))))